### PR TITLE
Allow match head powder for primer recipes

### PR DESCRIPTION
**Pull request by chaosvolt**
#### Summary

SUMMARY: Balance "Allow using match head powder as a source of impact-sensitive material for primer crafting"

#### Purpose of change

A while back I recall there having been discussion of using match heads as an option for making primers. Rationale being, if you mix the main material from matchheads with that of the striker surface (or the entirity of the matchhead, in the case of strike-anywhere matches), you get an impact-sensitive material that people have successfully tested using for pistol primers.

First video example I was able to find, evidently a reupload of a video I'm going to guess was the source of this idea: https://www.youtube.com/watch?v=CQI2bvkE7ic

#### Describe the solution

Added match head powder to primer recipes as an alternative to oxidizer powder, at equal amounts by weight.

#### Describe alternatives you've considered

1. Having the added amount equal oxidizer powder by volume would use 125 match head powder instead (100 oxidizer powder is 125 ml due to a count of 200 per 200-ml stack, while match head powder uses 10-count stacks with 10 ml volume).
2. Removing the oxidizer powder as an option outright. It's not very realistic but oxy powder seems to be generic anyway, and probably still useful as an alternative in the absence of any recipes for fulminates or azides, currently absent among the myriad of grandfathered-from-DDA chemicals that don't have any use yet.

#### Testing

Tested file change for linting and syntax.

#### Additional context

I don't recall how far the proposal originally got, but I'm going to guess it probably ran into problems due to matchbooks being a tool, so it would've had to displace the oxidizer powder's use as a component entirely to work right. With match head powder now being an explicit material that can be crafted, it's feasible to implement this idea.

R.I.P. Grant Thompson, original maker of the video linked above.

### CODE REVIEW

**Comment by Coolthulhu** [comment]
I'm not sure, it sounds more like a realism thing than gameplay thing.
What's the rationale?

**Comment by chaosvolt** [comment]
Lil bit realism, lil bit gameplay. Gameplay-wise, I'd say it's a combination of:
1. Matchhead powder presently has very little gameplay use, and making them a viable option as a source of primers gives them a niche that doesn't make them compete with other types of makeshift gunpowder.
2. Gives a mid-late game character an additional use for matchbooks, which they will likely have a hoard of sooner or later.
3. Makes oxidizer powder less of an RNG-dependent speed bump to handloading. In the long run players will probably obtain some oxy powder eventually, but in my experience thus far they seem scattered in itemgroups such that they suffer from Autoclave Syndrome.

So basically various reasons that allow craftable primers to be an option players are more likely to consider as an alternative to looting them or waiting until they have a bullet puller, whereas currently I don't think I've ever been in a situation where crafting my own primers was worth pursuing, in my experience so far.

**Comment by Coolthulhu** [comment]
> Makes oxidizer powder less of an RNG-dependent speed bump to handloading.

Makes sense. But I'd rather see it only being useful for pistol and maybe shotgun primers, not rifle.

**Comment by chaosvolt** [comment]
I can do that I suppose, sure.
